clients/fides-js/src/components/ConsentBanner.tsx

```diff
@@ -1,88 +1,39 @@
-import { h, FunctionComponent } from "preact";
-import { ButtonType, ExperienceConfig } from "../lib/consent-types";
-import Button from "./Button";
+import { h, FunctionComponent, VNode } from "preact";
+import { ExperienceConfig } from "../lib/consent-types";
 import CloseButton from "./CloseButton";
 
 interface BannerProps {
   experience: ExperienceConfig;
-  onAcceptAll: () => void;
-  onRejectAll: () => void;
-  onManagePreferences: () => void;
   onClose: () => void;
   bannerIsOpen: boolean;
+  buttonGroup: VNode;
```

Comment:
this pattern is known as react slots, and is handy for a few things. in this case, it's nice because `ConsentBanner` and `ConsentModal` both have these buttons that need to know how to save and a few other things. our component tree looks like

1. Overlay // knows how to save
2. ConsentModal / ConsentBanner // doesn't care about saving
3. ConsentButtons // needs to call save

so rather than prop drilling through the modal and banner, we can actually instantiate the grandchild in the parent element, and consolidate code that way